src/components/CodeBlock.vue

```diff
@@ -124,7 +124,6 @@ div[class*='language-']::before,
   position: absolute;
   top: 0;
   bottom: 0;
-  z-index: 3;
```

Comment:
Also, why these `z-index` removals ?

Reply:
During the rendering of this code block the terminal margin line appears in front of the Download Zip Success message and without removing z index, this problem could not be fixed